packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -43,6 +47,10 @@ export class ModalBase extends BaseComponent<IModalProps, IDialogState> implemen
       isVisible: props.isOpen,
       hasBeenOpened: props.isOpen
     };
+
+    this._warnDeprecations({
+      onLayerDidMount: 'layerProps.onLayerDidMount'
```

Comment:
This actually works on nested properties? I didn't know that!

Reply:
It only works if the deprecated property is not nested. The values of the `deprecationMap` are used for the deprecation warning text.